simplq/src/store/selectedQueueHistory/selectedQueueHistorySlice.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign */
+
+import { createSlice } from '@reduxjs/toolkit';
+import {
+  getSelectedQueueHistory,
+} from 'store/asyncActions';
+
+const selectedQueueHistorySlice = createSlice({
+  name: 'selectedQueueHistory',
+  initialState: {
+    events: [],
+  },
+  reducers: {},
+  extraReducers: {
+    // handle fulfilled request
```

Comment:
As a follow up PR, can you also subscribe to `joinQueue.fulfilled` and `deleteToken.fulfilled` events and update the events state by adding events to it?

Ref [here](https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/src/store/selectedQueue/selectedQueueSlice.js#L24)

Reply:
@daltonfury for that, the post response should be updated I think. Like currently the action payload for deleteToken:fulfilled request, consists of the following json, which doesn't consist of the expected eventTimestamp. I think the backend delete response should be updated so that to update as you are wanting

{
    "tokenId": "b3abb7b2-7a5e-4513-a511-796f48fe8c1a",
    "queueName": "Myq1",
    "tokenStatus": "REMOVED"
}

Reply:
@daltonfury42 , I can update things for joinQueue.fultilled request, by manipulating the incomming action.payload and taking the relevent fields. But one need to update the delete api response. Can you please see that and make the changes in the backend, so that I can update changes in the frontend